bin/vaultclient

```diff
@@ -12,6 +12,10 @@ const vaultclient = require('../index');
 
 const version = require('../package.json').version;
 
+const internalServiceAccountId = '000000000000';
+const internalServiceAccountName = 'scality-internal-services';
+const internalServiceAccountEmail = 'scality@internal';
```

Comment:
as `vaultclient` is public, would exposing the id this be a security risk in anyway?

Reply:
The id or name in themselves are of little-to-no value, and not making them public does not mean that someone will not leak them in the future. What needs to be hidden is credentials :)